udata/core/dataset/croquemort.py

```diff
@@ -64,8 +64,8 @@ def check_url_from_cache(url, group=None):
         response = requests.get(retrieve_url,
                                 params={'url': url, 'group': group},
                                 timeout=TIMEOUT)
-    except requests.ConnectionError:
-        return {'error': 'URL checker not found'}, {}
+    except requests.RequestException:
```

Comment:
I think that the `ConnectionError` is more specific here and that we should keep it to make the difference between the explicit checked not found and other errors.